src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-wizard/rgw-multisite-wizard.component.html

```diff
@@ -380,6 +381,13 @@
 </ng-template>
 
 <ng-template #reviewTemplate>
+  <cd-alert-panel type="warning"
+                  [showTitle]="false">
+    <span i18n>
+      During the automation process, the RGW module will be enabled on both the source and target clusters, if it is not already enabled.
+      This action may cause a temporary downtime (5-10 seconds) on each cluster.
```

Comment:
I think this can be the alert title itself and above sentence being description
Title being concise like : "Temporary downtime" or something better. Check with doc team.

Not a very strong opinion just a suggestion to consider.